client/store-management-links/quick-link/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React from '@wordpress/element';
+import { external, Icon } from '@wordpress/icons';
+import { __experimentalText as Text } from '@wordpress/components';
+import { Link } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export const QuickLink = ( { icon, title, href, linkType, onClick } ) => {
+	const isExternal = linkType === 'external';
+
+	return (
+		<div className="woocommerce-quick-links__item">
+			<Link
```

Comment:
Why isn't the `linkType` prop being passed to `<Link>`?

Reply:
Thanks for spotting this, just an error of omission. Will fix.

Reply:
I think for the links we're creating we should explicitly pass a `linkType` of "wp-admin" or "wc-admin". I doubt there's much to break with the regex method for detecting if an external/extension link could actually be a "wc-admin" type, but out of caution how about we force all external/extension links to be of type "wp-admin"?

Reply:
Sounds good. I'll ship that

Reply:
@jeffstieler I had another think about this and I want to break down my reasoning about the approach I'll take here based on your feedback.

There are several link types generated by the original quick links code:

`wc-settings`, `wc-admin`, `wp-admin` and `external`.

There are fewer link types respected by the `<Link>` component:

`wp-admin`, `wc-admin`, `external`

I think the right choice here is to ignore the fact that `wp-admin` and `wc-admin` links are not functionally different. Rather we should respect the API in case someone else decides that perhaps external links should behave differently for example.

So steps I'll take are:

1. Remove the `wc-settings` type from existing code. Replace it with `wp-admin` type
2. Normalise all `extension` links to `wp-admin` type
3. For remaining `wc-admin` links I will keep them as `wc-admin` type and pass the link type down to `<Link>`

I think this is similar to what you suggested but your suggestions were a little ambiguous so I've spelled it out here.

Reply:
@jeffstieler I had another think about this and I want to break down my reasoning about the approach I'll take here based on your feedback.

The link types we normalise to pass down to `<QuickLink>` right now include:

`wc-admin`, `wp-admin`, `external` and `extension`

There are fewer link types respected by the `<Link>` component:

`wp-admin`, `wc-admin`, `external`

I think the right choice here is to ignore the fact that `wp-admin` and `wc-admin` links are not functionally different. Rather we should respect the API in case someone else decides that perhaps external links should behave differently for example.

So steps I'll take are:

1. Remove the `extension` type from existing code. Replace it with `wp-admin` type
2. For remaining `wc-admin` links I will keep them as `wc-admin` type and pass the link type down to `<Link>`

I think this is similar to what you suggested but it felt a little ambiguous so I've spelled it out here.

Reply:
I've addressed this here: https://github.com/woocommerce/woocommerce-admin/pull/5476/commits/5b5af412fa8b47224f8d804cf1e3289f0b584a40

I'm going to merge this based on your pre-approval, but if you see any problems with the approach I've taken here then please let me know and I can address it in a follow up PR.